about.html

```diff
@@ -7,57 +7,95 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/aboutindex.css">
 
 </head>
 
+<header>
+  <div class="topstuff">
+   <img src="img/lambda-black.png" class="logo">
+
```

Comment:
If it fine to give each anchor their own class name, the nav class should be able to take care of everything and the links are the same so you could omit these unless you were doing something different to each <a> tag.